src/script/components/calling/ChooseScreen.tsx

```diff
@@ -17,23 +17,38 @@
  *
  */
 
-import React, {Fragment, useEffect} from 'react';
+import React, {Fragment, useCallback, useEffect} from 'react';
 
+import {container} from 'tsyringe';
+
+import {CallState} from 'src/script/calling/CallState';
+import {useKoSubscribableChildren} from 'Util/ComponentUtil';
 import {t} from 'Util/LocalizerUtil';
 
 export interface Screen {
   id: string;
-  thumbnail: HTMLCanvasElement;
+  thumbnail?: HTMLCanvasElement;
 }
 
 export interface ChooseScreenProps {
-  cancel: () => void;
   choose: (screenId: string) => void;
-  screens: Screen[];
-  windows: Screen[];
+  callState?: CallState;
 }
 
-const ChooseScreen: React.FC<ChooseScreenProps> = ({cancel, choose, screens = [], windows = []}: ChooseScreenProps) => {
+const ChooseScreen: React.FC<ChooseScreenProps> = ({
```

Comment:
You don't need there additional types
```suggestion
const ChooseScreen = ({
```

Reply:
because of lint error I applied the changes locally and pushed https://github.com/wireapp/wire-webapp/pull/17822/commits/bfbb157a5e1e729a6ac9f4c3f6c5b279fffbe3a3